pubspec.yaml

```diff
@@ -91,6 +91,10 @@ dependencies:
   wakelock_plus: ^1.1.3
   webrtc_interface: ^1.0.13
 
+  # Add for Tawkie
+  icons_plus: ^4.0.0
+  uuid: ^3.0.7
```

Comment:
I don't understand why we need to add a `uuid` lib, what does the app normally use when sending messages ?

Reply:
The application sends messages directly from the Room, so there's no need to provide a uid ourselves with this approach.

I hadn't yet found this method when I started the connection logic and I assumed that I had to send this type of message from the Client. But the Room method is more effective in preparing us for the use of multiple accounts on the same social network. I'll do changement